apps/site/pages/docs/go-live/overview.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: '  Overview'
+sidebar_label: 'Go-live'
+---
+
+import { Callout, Tab, Tabs } from 'nextra-theme-docs'
+import { CheckboxField } from '@faststore/ui'
+
+<header>
+
+# Go-live overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In a headless architecture, an ecommerce website relies on the seamless collaboration of various modules. FastStore specializes in the frontend aspect, enabling the development of high-performance user interfaces for ecommerce websites. However, to create a complete digital commerce experience, multiple core modules must be integrated. At **VTEX**, some of these modules are:
+
+| Module                      | Description                                                                                                                             |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| **Checkout**                | Receives the shopper's information necessary to process an order.                                                                       |
+| **OrderPlaced**             | Handles the processing of customer orders. It displays a successful message right after the checkout process is completed.                                                            |
```

Comment:
Isn't the processing part of the checkout?

Reply:
Yes, Guila!